static/src/stylesheets/module/facia/item-tones/_tone-live.scss

```diff
@@ -80,3 +80,53 @@
 .tone-live--most-popular .fc-item__kicker {
     color: colour(live-accent);
 }
+
+.fc-item__liveblog-blocks {
+    height: 82px;
+    overflow: hidden;
+}
+
+.fc-item__liveblog-blocks__inner {
+    transition: transform .5s ease-in-out;
+}
+
+.fc-item__liveblog-block {
+    display: block;
+    position: relative;
+    @include fs-headline(1);
+    color: #ffffff;
+    overflow: hidden;
+    height: $gs-baseline * 3 + 2;
+    margin-bottom: $gs-baseline / 2;
+    transition: height .8s linear;
+}
+
+.fc-item__liveblog-block__more {
+    position: absolute;
+    bottom: $gs-baseline / 6;
+    right: 0;
+    width: $gs-gutter;
+    padding-left: $gs-gutter / 10;
+    background-color: colour(live-default);
+}
+
+.u-faux-block-link--hover .fc-item__liveblog-block__more {
```

Comment:
shoudl probably pin this class to `faux` bit on line 11 (`&`, `& .fc-item__liveblog-block__more`)